### ISSUE TITLE
Query failed. Please inspect Grafana server log for details

### ISSUE THREAD

**Issue by fe-linjin** [grafana/grafana#24608]
**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by ivanahuckova** [comment]
Sorry, but not enough details to look into this. If you would like to reopen the issue, please provide more information.
